File: javascript/gemini/p62.js
const fs = require('fs');
const data = fs.readFileSync('input.dat');
let checksum = 0;
for (const byte of data) checksum += byte;

if (checksum === 0) console.log("Empty");

fs.writeFileSync('stats.txt', `Bytes: ${data.length}, Checksum: ${checksum}`);

File: python/gemini/p62.py
def process_file():
    with open("input.dat", "rb") as f:
        data = f.read()

    checksum = sum(data)
    if checksum == 0:
        print("Buffer is empty")

    with open("stats.txt", "w") as f:
        f.write(f"Bytes: {len(data)}, Checksum: {checksum}")
